services/syncfusion-filesystem-provider/ej2-amazon-s3-aspcore-file-provider/appsettings.json

```diff
@@ -0,0 +1,12 @@
+{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Warning"
+    }
+  },
+  "AllowedHosts": "*",
+  "Jwt": {
```

Comment:
As I was mentioning during standup this morning, I think these might as well be moved to our .env and then read the same way as all other environment variables.